hera_cal/utils.py

```diff
@@ -23,19 +23,40 @@
     AIPY = False
 
 
+# Defines characters to look for to see if the polarization string is in east/north format. Nominally ['e', 'n'].
+_KEY_CARDINAL_CHARS = [c.lower() for c in _x_orientation_rep_dict('north').values()]
+
+
+def _is_cardinal(polstr):
+    '''Returns true if any characters in the polarization match those in _KEY_CARDINAL_CHARS.'''
+    return np.any([c.lower() in _KEY_CARDINAL_CHARS for c in polstr])
```

Comment:
this worries me a bit. I can think of a million silly ways this can break, but a couple important ones are, "unknown" or "none". Is it not possible to just list all the possible (lower case) cardinal versions?

Reply:
That's a fair point. I'm trying to explicitly define everything in terms of the hard-coded definitions in pyuvdata, so that if those change again we might still be OK. Let me think about this for a bit and maybe I can come up with something more robust.

Reply:
Does 41b5162 satisfy @adampbeardsley ?

Reply:
Yeah, I think that works. Thanks!